build/steps-build.yml

```diff
@@ -39,7 +58,16 @@
     msbuildLocationMethod: version
     msbuildVersion: latest
     msbuildArchitecture: x86
-    msbuildArguments: /p:PackageVersion=$(SemVer) /p:ApplicationEnvironment=$(ApplicationEnvironment) /p:ApplicationVersion=$(MajorMinorPatch) /p:AndroidSigningKeyStore=$(keyStore.secureFilePath) /p:AndroidSigningStorePass=$(AndroidSigningStorePass) /p:AndroidSigningKeyPass=$(AndroidSigningKeyPass) /p:AndroidSigningKeyAlias=$(AndroidSigningKeyAlias) /p:AndroidKeyStore=true
+    msbuildArguments: >
+      /p:PackageVersion=$(SemVer)
+      /p:ApplicationEnvironment=$(ApplicationEnvironment)
+      /p:ApplicationVersion=$(MajorMinorPatch)
+      /p:AndroidSigningKeyStore=$(keyStore.secureFilePath)
+      /p:AndroidSigningStorePass=$(AndroidSigningStorePass)
+      /p:AndroidSigningKeyPass=$(AndroidSigningKeyPass)
+      /p:AndroidSigningKeyAlias=$(AndroidSigningKeyAlias)
+      /p:AndroidKeyStore=true
+      /p:CodesignProvision=$(provisioningProfile.provisioningProfileUuid)
```

Comment:
If you want to make it explicit where the ApplicationIdentifier is used, you could add `/p:ApplicationIdentifier=$(ApplicationIdentifier)` here. Though I'm not 100% sure it would behave properly if it isn't set...

Reply:
So I should just set it then :D ?